src/scripts/r_config.sh

```diff
@@ -35,7 +35,7 @@ if test -z "$CXX"; then
 fi
 
 # Only R > 4 will have the variables
-${R_HOME}/bin${R_ARCH_BIN}/Rscript --vanilla -e "getRversion() > 4" | grep TRUE > /dev/null
+${R_HOME}/bin${R_ARCH_BIN}/Rscript --vanilla -e "getRversion() > '4.0.0'" | grep TRUE > /dev/null
```

Comment:
Decades of shell experience makes me fear I _really_ prefer out single and inner double here.  Can you revert, please?